test/test_tensor_creation_ops.py

```diff
@@ -2668,6 +2668,15 @@ def test_empty_tensor_props(self, device):
             y = torch.empty(tuple(size_ones_instead_of_zeros), device=device)
             self.assertEqual(x.stride(), y.stride())
 
+    @onlyNativeDeviceTypes
+    def test_empty_overflow(self, device):
+        with self.assertRaisesRegex(RuntimeError, 'Storage size calculation overflowed'):
+            torch.empty([2, 4, 536870912, 536870912], dtype=torch.float64)
```

Comment:
Nit, please use hex or 1<<29 or 2**29, which makes this check much more readable
```suggestion
            torch.empty([2, 4, 2**29, 2**29], dtype=torch.float64)
```